hermes/bin/tests/wasm-integration/main.rs

```diff
@@ -23,101 +23,132 @@ use hermes::{
     wasm::module::Module,
 };
 use libtest_mimic::{Arguments, Failed, Measurement, Trial};
+use tracing::{level_filters::LevelFilter, subscriber::SetGlobalDefaultError};
+use tracing_subscriber::{fmt::time, FmtSubscriber};
+
+/// Init the logger
+fn init_logger() -> Result<(), SetGlobalDefaultError> {
+    let subscriber = FmtSubscriber::builder()
+        .json()
+        .with_level(true)
+        .with_thread_names(true)
+        .with_thread_ids(true)
+        .with_file(true)
+        .with_line_number(true)
+        .with_timer(time::UtcTime::rfc_3339())
+        //.with_span_events(FmtSpan::CLOSE)
```

Comment:
Can you please explain why you chose to comment out this line of code instead of deleting it?

Reply:
its just an artifact left over from testing.